__tests__/ships.test.js

```diff
@@ -0,0 +1,77 @@
+
+const Ship = require('../src/ships');
+
+let dover;
+let itinerary;
+let ship;
+let calais;
+
+describe('Ship constructor', () => {
+    describe('with ports and an itinerary', () => {
+        beforeEach(() => {
```

Comment:
Same with this beforeEach. Since this is a method you're using in each test, for safety, I would have it as up in the scope as possible!